biojava-structure/src/main/java/org/biojava/nbio/structure/rcsb/RCSBUpdates.java

```diff
@@ -0,0 +1,60 @@
+package org.biojava.nbio.structure.rcsb;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.URL;
+import java.net.URLConnection;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class RCSBUpdates {
+
+
+	public Map<String,String[]> getUpdates(){
```

Comment:
Could you add some minimal javadoc here? It is not obvious what the return map contains